anymail/backends/mailpace.py

```diff
@@ -0,0 +1,190 @@
+from ..exceptions import AnymailRequestsAPIError
+from ..message import AnymailRecipientStatus
+from ..utils import CaseInsensitiveCasePreservingDict, get_anymail_setting
+from .base_requests import AnymailRequestsBackend, RequestsPayload
+
+
+class EmailBackend(AnymailRequestsBackend):
+    """
+    MailPace API Email Backend
+    """
+
+    esp_name = "MailPace"
+
+    def __init__(self, **kwargs):
+        """Init options from Django settings"""
+        esp_name = self.esp_name
+        self.server_token = get_anymail_setting(
+            "server_token", esp_name=esp_name, kwargs=kwargs, allow_bare=True
+        )
+        api_url = get_anymail_setting(
+            "api_url",
+            esp_name=esp_name,
+            kwargs=kwargs,
+            default="https://app.mailpace.com/api/v1/send",
+        )
+        if not api_url.endswith("/"):
+            api_url += "/"
+        super().__init__(api_url, **kwargs)
+
+    def build_message_payload(self, message, defaults):
+        return MailPacePayload(message, defaults, self)
+
+    def raise_for_status(self, response, payload, message):
+        # We need to handle 400 responses in parse_recipient_status
+        if response.status_code != 400:
+            super().raise_for_status(response, payload, message)
+
+    def parse_recipient_status(self, response, payload, message):
+        # Prepare the dict by setting everything to queued without a message id
+        unknown_status = AnymailRecipientStatus(message_id=None, status="queued")
```

Comment:
Shouldn't this be `status="unknown"`? (And then it gets overwritten below for recipients whose status is known.)